src/main/kotlin/network/warzone/mars/player/achievements/gui/AchievementMenu.kt

```diff
@@ -0,0 +1,210 @@
+package network.warzone.mars.player.achievements.gui
+
+import net.md_5.bungee.api.ChatColor
```

Comment:
small nitpick but could you import ChatColor from `org.bukkit` instead as that's what the rest of the codebase uses